bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -139,6 +160,39 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
         }
     }
 
+    private Collection<Resource> getAssetFolders(Page page, ResourceResolver resolver) {
```

Comment:
Responded to the ResourceVisitor refactor on https://github.com/HS2-SOLUTIONS/acs-aem-commons/pull/2.  Let's continue the conversation there.